src/components/CustomComponent.jsx

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+export default class CustomComponent extends React.Component {
```

Comment:
For future - please avoid using default exports. It will be hard to add something else to export. 

Just `export class CustomComponent ...` will be enough

Reply:
Fixed